build.gradle.kts

```diff
@@ -308,21 +413,22 @@ subprojects {
     }
 
     tasks.withType<JavaCompile>().configureEach {
+        options.compilerArgs.add("-Xlint:deprecation")
```

Comment:
Any particular reason to add this?

Reply:
I was wondering what it might show in relation to Gradle deprecration itself but I don't think it does. I will remove it.